sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/EventDataBatchIntegrationTest.java

```diff
@@ -83,7 +83,7 @@ public void sendSmallEventsFullBatch() {
     public void sendSmallEventsFullBatchPartitionKey() {
         // Arrange
         final EventDataBatch batch = new EventDataBatch(ClientConstants.MAX_MESSAGE_LENGTH_BYTES, null, PARTITION_KEY, contextProvider,
-            new TracerProvider(Collections.emptyList()));
+            new TracerProvider(Collections.emptyList()), null, null);
```

Comment:
Instead of using null, null.. can you use getFullyQualifiedDomainName() and getEventHubName() ? These properties are set in IntegrationTestBase.

Reply:
updated.